force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1697,6 +1697,15 @@
         <shortDescription>Error for DIs with populated RD fields when RD2 is disabled.</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can process Recurring Donations.</value>
     </labels>
+    <labels>
+        <fullName>RD2_nextDonation</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for next donation mapping.</shortDescription>
+        <value>Paid Installments: {0}/{1},
+Next Installment Date: {2}.</value>
```

Comment:
I would still prefer two separate labels. Best practice is that we do not escape labels when displaying. Generally embedded a CRLF into the label may not render as expected.

Reply:
changes applied!